core/storage/config/gae_models.py

```diff
@@ -25,6 +25,8 @@
 
 MYPY = False
 if MYPY: # pragma: no cover
+    # Here, we are importing 'platform_parameter_domain' only for type checking.
+    from core.domain import platform_parameter_domain  # pylint: disable=invalid-import # isort:skip
```

Comment:
Above you also import in the if-block and there is no explanatory comment, should it be added above?

Reply:
Also, why are the ignores needed?